app/controllers/users/two_factor_authentication_setup_controller.rb

```diff
@@ -4,7 +4,7 @@ class TwoFactorAuthenticationSetupController < ApplicationController
     include MfaSetupConcern
 
     before_action :authenticate_user
-    before_action :confirm_user_authenticated_for_2fa_setup
+    before_action :confirm_user_needs_2fa_setup
```

Comment:
`confirm_user_authenticated_for_2fa_setup` is meant to authenticate that a user can setup MFA. It is meant for re-entrant things like phone and TOTP setup. Since we are dropping the backup MFA requirement this screen should not be re-entrant. I replaced `confirm_user_authenticated_for_2fa_setup` with a method that gives us what we want